File: examples-sdk/zig/204-llm-streaming-translator/main.zig
// 204-llm-streaming-translator — Zig SDK equivalent
// Compile: vil compile --from zig --input 204-llm-streaming-translator/main.zig --release

const vil = @import("vil");

pub fn main() void {
    var server = vil.Server.init("llm-streaming-translator", 3103);
    var translator = vil.Service.init("translator");
    server.service(&translator);
    server.compile();
}

File: examples-sdk/go/204-llm-streaming-translator/main.go
// 204-llm-streaming-translator — Go SDK equivalent
// Compile: vil compile --from go --input 204-llm-streaming-translator/main.go --release
package main

import vil "github.com/OceanOS-id/vil-go"

func main() {
	s := vil.NewServer("llm-streaming-translator", 3103)

	translator := vil.NewService("translator")
	s.Service(translator)

	s.Compile()
}
